### ISSUE TITLE
Cannot open related review in a separate tab

### ISSUE THREAD

**Issue by heaths** [Azure/azure-sdk-tools#2813]
To more easily compare APIs for different languages, it would be efficient to middle-click / open in new tab a link from the drop-down. However, since it's not an actual link, you can't do this. Instead, I have to copy the URL into a new tab, then select a new language.

**Comment by maririos** [comment]
Testing in current instance suggest this is working. @heaths could you confirm in case I didn't fully understand the scenario?

**Comment by heaths** [comment]
Yes, from what I remember - and if I take the request literally - this works now.
